wezterm-gui/src/overlay/copy.rs

```diff
@@ -822,6 +822,54 @@ impl CopyRenderable {
         }
     }
 
+    fn move_backward_one_word_start(&mut self) {
```

Comment:
It's hard to tell at a glance just how similar this is to the `move_backward_one_word` function. If adding a parameter to control whether it should be in "start mode" or not makes for a smaller diff and a more readable function, that would get my vote!